packages/components/package.json

```diff
@@ -58,7 +58,6 @@
 		"i18n-calypso": "workspace:^",
 		"lodash": "^4.17.21",
 		"prop-types": "^15.7.2",
-		"qrcode.react": "^3.1.0",
```

Comment:
This is already a dependency of Calypso's client, but is no longer a dependency of `@automattic/components`.